source/src/core/simple_metrics/metrics/RMSDMetric.hh

```diff
@@ -130,6 +131,22 @@ public:
 	set_run_superimpose( bool super );
 
 	///@brief Set what we will be calculating the RMSD on.
+	/// By default RMSD is calcualted between all atoms other than hydrogens (all_heavy).
+	///
+	/// You can look up core/rms/rms_enum.hh for available options. Right now the available options are:
+	/// rmsd_protein_bb_heavy: backbone heavy atoms N,CA,C 
+	/// rmsd_protein_bb_heavy_including_O: backbone heavy atoms N,CA,C,O 
+	/// rmsd_protein_bb_ca: CA (C-alpha)
+	/// rmsd_sc_heavy: side chain heavy atoms
+    /// rmsd_sc: all side chain atoms (including H)
```

Comment:
Spacing?

Reply:
between lines?

Reply:
Sorry, I meant "indentation" rather than spacing. The code fails the [beautification](https://b3.graylab.jhu.edu/test/833703) test, due to the inconsistent indentation.

Reply:
no worries. I corrected it in the new commit.